packages/react-image/Spec.md

```diff
@@ -0,0 +1,161 @@
+# @fluentui/react-image
+
+## Background
+
+Image is a visual representation of some content, such as a photo, illustration etc.
+
+## Concerns
+
+Based on the OpenUI research and the Figma specification, Image seems to be one of those components that is purely styling, thus I would appreciate your answers in any of these questions:
+
+- Are there any components would need to use Image?
+- Apart from styling, are there any other functionalities for Image?
+- Is there any benefit of having Image as a component? (taking into consideration that it is css styling and we will need to maintain it over time.)
```

Comment:
Notes from discussion within @microsoft/cxe-prg:

Currently `Image` doesn't do much, besides providing `styling` abstraction, which could/should be covered by proper design tokens/makeStyles.

Another case that is not covered by this component is when user want to use `figure` with appropriate `source`/`img` fallbacks. (to cover this we would again go into unnecessary abstraction)


**What value should `Image` abstraction provide to consumers:**
- we would need to do something similar like [nextjs/image](https://nextjs.org/docs/api-reference/next/image). 
- for us that's out of scope as we don't control consumers build pipelines etc
 

**Outcome:**
- we don't think this component should be shipped as is
- what would make sense is to provide some aggregated style variants (similar to what react-text does), IFF  that's not already covered by design-tokens

Reply:
The other value Image has provided in the past is sizing helpers (how to fit/scale the image *in the browser*--modifying images in build pipelines is definitely out of scope). Is that no longer a priority, or do you think it should be covered by reusable styling helpers or something instead of a component?

Reply:
@ecraig12345 IMO a component should have actual functionality/behaviour rather than only styling abstractions, in this case we could offer styling helpers (like typographyStyles in react-text), even though these styles are quite simple and easily achievable with makeStyles. Check the Image styles [here](https://github.com/microsoft/fluentui/blob/master/packages/react-image/src/components/Image/useImageStyles.ts).

Reply:
These questions were addressed during the Tech Sync and the spec is updated with the context/results from the meeting. 
The main conclusion was the Image will remain as a component, initially it will contain styling and will be iterated for features/functionalities when needed in the future.